sous_chef/blueprints.py

```diff
@@ -45,6 +45,12 @@ def get_chef_environments():
     """Ensure a list of environments is availible"""
     flask.g.chef_environments = flask.current_app.chef.get('environments')
 
+    # Set the default organization with a sensible default
```

Comment:
I'm not sure this is the correct place for this, but it's the only place I could get it working